web/docs/migration-guides/migrate-from-0-19-to-0-20.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Migration from 0.19.X to 0.20.X
+---
+
+To install Wasp 0.20.0 on Linux / OSX / WSL (Windows), open your terminal and run:
+```sh
+curl -sSL https://get.wasp.sh/installer.sh | sh
+```
```

Comment:
This is something new I've added because I think it's useful.

Reply:
Should we specify the 0.20.0 version argument?